test/Core.Test/Services/PolicyServiceTests.cs

```diff
@@ -446,6 +446,50 @@ public async Task GetPoliciesApplicableToUserAsync_WithDisableSendTypeFilter_Wit
         Assert.True(result.All(details => !details.IsProvider));
     }
 
+    [Theory, BitAutoData]
+    public async Task AnyPoliciesApplicableToUserAsync_WithRequireSsoTypeFilter_WithDefaultUserTypeFilter_WithDefaultOrganizationUserStatusFilter_ReturnsFalse(Guid userId, SutProvider<PolicyService> sutProvider)
```

Comment:
suggestion: these test names are too focused on inputs and outputs and do not describe the actual behaviour being tested. What's the general idea here? "returns false if policy does not apply to user"? "RequireSSO policy does not apply to Owners"? I know we have to jam it into a method name so it won't look exactly like that (I prefer Jest for this reason 😁 ) but you get the idea.

Reply:
This is non-blocking, a suggestion only.